.github/workflows/test.yml

```diff
@@ -17,15 +17,16 @@ jobs:
         tarantool-version: [2.6, 2.7]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: update git submodules
-      run: git submodule update --init --recursive
+    - uses: actions/checkout@v3
+      with:
```

Comment:
Shouldn't it be a part of refactoring? Looks like 

```yaml
with:
  fetch-depth: 1
  submodules: recursive
```

should go to another commit.

Reply:
fixed.